.github/workflows/renovate-ai-analysis.yaml

```
@@ -0,0 +1,111 @@
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+    branches: [main]
+
+jobs:
+  ai-analysis:
+    permissions:
+      contents: read
+      pull-requests: write
+      models: read
```

actionlint (1.7.7)

unknown permission scope "models". all available permission scopes are "actions", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)